examples/tensorflow_simple.py

```diff
@@ -0,0 +1,148 @@
+"""
+Optuna example that optimizes multi-layer perceptrons using Tensorflow.
+
+In this example, we optimize the validation accuracy of hand-written digit recognition using
+Tensorflow and MNIST. We optimize the neural network architecture as well as the optimizer
+configuration. As it is too time consuming to use the whole MNIST dataset, we here use a small
+subset of it.
+
+We have the following two ways to execute this example:
+
+(1) Execute this code directly.
+    $ python tensorflow_simple.py
```

Comment:
Because this example is specific to estimator interface, let's rename the filename to `tensorflow_estimator.py`.

Reply:
We decided to rename `pruning/tensorflow_integration.py` to `pruning/tensorflow_estimator_integration.py`.